src/Sia.Gateway/Controllers/Playbook/EventTypesController.cs

```diff
@@ -22,10 +22,14 @@ public EventTypesController(IMediator mediator, AzureActiveDirectoryAuthenticati
 
         [HttpGet(Name = nameof(GetAll) + nameof(EventType))]
         public async Task<IActionResult> GetAll()
-            => Ok(await _mediator.Send(new GetEventTypesRequest(_authContext)));
+        {
+            return OkIfFound(await _mediator.Send(new GetEventTypesRequest(_authContext)));
```

Comment:
For single line functions use the expression syntax; this diff should only be changing Ok to OkIfFound.